docs/_docs/installation/windows.md

```diff
@@ -60,7 +60,7 @@ sudo apt-get install ruby2.5 ruby2.5-dev build-essential dh-autoreconf
 Next let's update our Ruby gems:
 
 ```sh
-gem update
+sudo gem update
```

Comment:
The install step below is done _without_ sudo, therefore this should also exclude it.

In fact, if you need permissions errors, there are at least two solutions without using `sudo`:

- Change the permissions of the shared gems folder (`/usr/.../...`) to be editable by all users.
- Use `gem update --user-install` and `gem install GEM_NAME --user-install` to install to the user's `~/.gem` directory.

This is a complex topic - in fact `--user-install` may become an implied optional in newer `gem` command.

So it is best to leave this as it was.

Reply:
```
gem update --help

...
        --[no-]user-install          Install in user's home directory instead
                                     of GEM_HOME.
```